curriculum/challenges/english/02-javascript-algorithms-and-data-structures/object-oriented-programming/understand-the-immediately-invoked-function-expression-iife.english.md

```diff
@@ -33,7 +33,7 @@ tests:
   - text: The function should be anonymous.
     testString: assert(/\((function|\(\))(=>|\(\)){?/.test(code.replace(/\s/g, "")));
   - text: Your function should have parentheses at the end of the expression to call it immediately.
-    testString: assert(/}?\)\(\)$|}\(\)\)$/.test(code.replace(/[\s;]/g, "")));
+    testString: assert(/\(.*\)\(\)|\(.*\}\(\)\)/.test(code.replace(/[\s;]/g, "")));
```

Comment:
```suggestion
    testString: assert(/\(.*(\)\(|\}\(\))\).test(code.replace(/[\s;]/g, "")));
```

Reply:
@JoshuaPelealu What do you think about this slightly more DRY version of your regex?

Reply:
I like it. It's more structured.